update-docs.sh

```diff
@@ -3,4 +3,6 @@
 set -euo pipefail
 
 bazel build //docs/...
-cp bazel-bin/docs/*.md docs/
\ No newline at end of file
+
+# Copy the file, ignoring the permissions inside bazel-bin
+cat bazel-bin/docs/APIReference.md > docs/APIReference.md
```

Comment:
Inside bazel-bin files are executable and not writable which is annoying.

What this does exactly depends on your umask, but git only tracks the most basic info about file permissions so in general this should be good.